test/NVQPP/integration/bug_qubit.cpp

```diff
@@ -11,27 +11,31 @@
 // RUN: nvq++ --enable-mlir -v %s --target quantinuum --emulate -o %t.x && %t.x | FileCheck %s
 // RUN: cudaq-quake %s | cudaq-opt --promote-qubit-allocation | FileCheck --check-prefixes=MLIR %s
 
-// CHECK: Test: { 0:{{[0-9]+}} 1:{{[0-9]+}} }
-
 #include <cudaq.h>
 #include <iostream>
 
-struct ak2 {
+struct simple_x {
   void operator()() __qpu__ {
     cudaq::qubit q;
-    h(q);
+    x(q);
+
+// FIXME: This doesn't really depend on us. IONQ's doesn't accept measurements.
```

Comment:
There are things that we can and should do on our end as well. I filed an issue to track that here: https://github.com/NVIDIA/cuda-quantum/issues/512
Please link that issue instead in all of these comments.
```suggestion
// TODO: Extend measurement support for submissions to IonQ, 
// see https://github.com/NVIDIA/cuda-quantum/issues/512.
```